### DISCUSSION TITLE
Background Color Transparency

### DISCUSSION THREAD

**Discussion by Jkraje** [Hofknecht/SystemTrayMenu#509] [Ideas]
Hey,

I'm so glad I found SystemTrayMenu. You do wonderful work there. It works like a charm and increases my workflow importantly. Thank you!

So I tried to use a transparent background like "#66ffffff" but than it crashes. Is it possible to use transparent backgrounds? Its just an optical candy, but I thought I just ask for it :)

Thank you very much!

Best Regards
Jonas

**Comment by Hofknecht** [comment]
Hey @Jkraje 
Thank you for the feedback!

in the code you could change
private const double Shown = 1.00
to;
private const double Shown = 0.90;

![image](https://github.com/Hofknecht/SystemTrayMenu/assets/52528841/b3bb37f4-e241-4347-b0b1-a078673f914a)

maybe we should consider a option therefore or maybe in wpf we can do Acrylic Window Background

**Reply by Jkraje** [reply]
Oh, that's great!
I will try as soon as possible.
An option therefore would be really cool as well.

Thank you very much and have a nice day.

Oder anders gesagt: Schönes Wochenende aus Schleswig-Holstein ;)

**Comment by Jkraje** [comment]
Hey :)

I just had time to try it out and it works. The thing is that not only the background is acrylic also the icons etc. only fade in 90%.
So I'm thinking about another solution. But thanks so far.
And great work btw!

Bye
